pym/portage/dep/__init__.py

```diff
@@ -239,147 +238,6 @@ def strip_empty(myarr):
 		('portage.dep.strip_empty',), DeprecationWarning, stacklevel=2)
 	return [x for x in myarr if x]
 
-def paren_reduce(mystr, _deprecation_warn=True):
```

Comment:
This one's still used by gentoolkit. If we really want to remove something then we need to make the deprecation warning unconditional (it currently only triggers for internal callers).

Reply:
I've seen it in qa-scripts as well. @gentoo/qa.